client/src/common/components/Activities/model/actionTypes.js

```diff
@@ -0,0 +1,4 @@
+export const ActivityActionTypes = Object.freeze({
+  FETCH_SUCCESS: 'activity/FETCH_SUCCESS',
+  FETCH_FAILURE: 'activity/FETCH_FAILURE'
```

Comment:
shouldn't we have plural here - `activities`? This fetches all activities, not only one?